.tekton/basic-component-pipeline.yaml

```diff
@@ -291,11 +448,11 @@ spec:
   - name: deprecated-base-image-check
```

Comment:
These EC findings are interesting

![image](https://github.com/user-attachments/assets/c2fda002-d178-421a-9556-5518f25a8a3c)

See https://console.redhat.com/preview/application-pipeline/workspaces/rh-acs/applications/acs/pipelineruns/acs-pk7q4-vbvw2/security

Since giving it just `-amd64` wasn't sufficient, I suppose we'll need to make the full matrix - do the deprecated base image check for each arch.

Reply:
I think I have an alternative for this, we can feed the manifest URL and digest, the base image has the same digests for all archs, so it should scan check all archs (at least that's how I have it in #12055 and I think it worked but now I can't find the pipeline).

Reply:
> the base image has the same digests for all archs

This is the base image manifest may have the same digest because it's just one manifest. As it's a manifest it will point to the actual images and those will have different digests.
Anyway, I'm interested in your approach and how would it work out.

Reply:
Turns out when using the v0.2 buildah steps with the v0.4 version of this task, it can use the SBOM directly to scan images, so we can remove `BASE_IMAGE_DIGESTS` altogether.